test/disposable/dispose-test.js

```diff
@@ -58,16 +58,15 @@ describe('tryDispose', function () {
     var spy = disposableSpy(returns, x)
 
     var result = dispose.tryDispose(0, spy, failSink)
-    expect(result).toBe(x)
+
+    eq(result, x)
   })
 
   it('should return disposable result promise', function () {
     var x = {}
     var spy = disposableSpy(returns, Promise.resolve(x))
 
-    return dispose.tryDispose(0, spy, failSink).then(function (y) {
-      expect(y).toBe(x)
-    })
+    return dispose.tryDispose(0, spy, failSink).then(eq(x))
```

Comment:
Many of these tests were designed to verify the object identity, so that the function under test can't invent an equivalent value and pass the test.  I'm not really convinced it matters (and in a truly immutable environment, it wouldn't!), but wanted to see what you think.  I can see a couple options:

1. Use `is`
2. Use `eq`, but use a value that is unlikely ever to cause a false positive, like `Math.random()` instead of `{}`

Thoughts?